docs/src/components/CodePreview/CodePreview.component.tsx

```diff
@@ -117,11 +129,16 @@ const StyledLivePreview = styled(LivePreview)<PreProps>`
     border-bottom-right-radius: 0.25rem;
 `;
 
+const StyledCodeBlock = styled('pre')`
+    padding: 1.25rem;
+    background-color: ${colors.silver.base};
+`;
+
 interface CodePreviewProps {
     children?: any;
     className?: string;
     live?: boolean;
```

Comment:
I'm wondering if this could be an enum of types instead of having a prop for each type?